core/src/components/chip/chip.common.scss

```diff
@@ -0,0 +1,36 @@
+@import "../../themes/mixins";
+
+// Chip: Common Styles
+// --------------------------------------------------
+
+:host {
+  /**
+   * @prop --background: Background of the chip
+   * @prop --color: Color of the chip
+   * @prop --border-radius: Border radius of the chip
```

Comment:
I don't see `--border-radius` being used in this file. Does this line still need to be here?

Reply:
Remembered that it's in here because it's being used in all the themes.